src/firefly/java/edu/caltech/ipac/firefly/server/servlets/AnyFileDownload.java

```diff
@@ -202,13 +198,10 @@ private void sendFileToClient(HttpServletRequest req, HttpServletResponse res, F
 
 
 
-        if (track) trackProgress(req, BackgroundEnv.DownloadProgress.WORKING);
         try {
             FileUtil.writeFileToStream(f, res.getOutputStream());
-            if (track) trackProgress(req, BackgroundEnv.DownloadProgress.DONE);
             if (log) logActivity(f);
         } catch (IOException e) {
-            if (track) trackProgress(req, BackgroundEnv.DownloadProgress.FAIL);
```

Comment:
`download(url, filename) in util/fetch.js:81` returns a promise.  I only track it on the client side. `core/background/BackgroundMonitor.jsx:221`.  
I did not send this flag to the server's JobInfo because I view it only as a UI feature.  If you reload the browser, the flag (checked mark) will go away.  If this is an issue, I can send the flag.